.eslintrc

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I would suggest copying the eslint rules that we created for tC minus the react stuff